packages/apps/marmalade-marketplace/src/components/MintToken/index.tsx

```diff
@@ -34,7 +36,7 @@ function MintTokenComponent() {
   
   const onTransactionSigned = (transaction: IUnsignedCommand | ICommand) => {
     setTransaction(transaction);
-    router.push(`/transaction?returnUrl=/tokens/${tokenId}`);
+    router.push(`/transaction?returnUrl=/tokens/${tokenId}?chainId=${`8`}`);
```

Comment:
is the `8` a preparation for it to be parameterized?

Reply:
Yes, user will be selecting the chainId, but we're setting it to 8 for now